cmd/yggd/main.go

```diff
@@ -358,6 +385,12 @@ func mainAction(c *cli.Context) error {
 		return err
 	}
 
+	fmt.Println("***")
+	err = setupFactsFile()
+	if err != nil {
+		return err
```

Comment:
Same thing wrapping the error message here.

Reply:
I was not sure whether the error message should be wrapped here. Some calls from `mainAction` do,

```go
	client, transporter, err := setupClient(dispatcher, tlsConfig)
	if err != nil { return cli.Exit(fmt.Errorf("cannot setup client: %w", err), 1) }
```

some don’t.


```go
	err = setupClientID()
	if err != nil { return err }
```

I think it’s good practice to wrap and there are already wrapped ones. I put the wrapper there, then. It allows to add a single `cli.Exit()` wrapper too.

I’ll create a card for consolidating the error returns.